python-app-dependencies/pom.xml

```diff
@@ -16,27 +16,37 @@
 		<relativePath/>
 	</parent>
 
+	<properties>
+		<spring-cloud-starter-stream.version>2.1.0.BUILD-SNAPSHOT</spring-cloud-starter-stream.version>
+		<spring-cloud-config-server.version>2.0.2.RELEASE</spring-cloud-config-server.version>
```

Comment:
Doesn't `spring-cloud-config-server` come from the Spring Cloud BOM?
Why do we need this explicit version here?